lib/model/seed_gen_model.dart

```diff
@@ -0,0 +1,18 @@
+class SeedGenModel {
```

Comment:
**question:** could this class be immutable?

Reply:
yes, It could be